tests/unit/amo/helpers.js

```diff
@@ -58,3 +65,20 @@ export const signedInApiState = Object.freeze({
   ...coreSignedInApiState,
   clientApp: 'firefox',
 });
+
+export function dispatchSignInActions({
```

Comment:
❤️ ❤️ ❤️ !

This probably won't entirely close #2494 because we'd need to get every test using it but that's awesome.